### ISSUE TITLE
Cannot invoke method containsKey() on null object when "igenomes_ignore=true"

### ISSUE THREAD

**Issue by mlorthiois** [nf-core/nanoseq#76]
Hi,

When I use the profile provide by my institution (Genouest), which set the parameter `igenomes_ignore = true`, I get this error :

```
Cannot invoke method containsKey() on null object

 -- Check script '/home/genouest/cnrs_umr6290/mlorthiois/.nextflow/assets/nf-core/nanoseq/main.nf' at line: 290 or see '.nextflow.log' file for more details
```

Solved by adding `--igenomes_ignore false` in my command line.

**Comment by drpatelh** [comment]
Ok. I can reproduce this locally. Let me see if I can find a fix.

**Comment by drpatelh** [comment]
This should be fixed now in #89 . Thanks for reporting @mlorthiois ! It may turn out that this was related to [#77](https://github.com/nf-core/nanoseq/issues/77) so may be worth trying the `dev` branch after that PR has been merged.
